### DISCUSSION TITLE
API module with HttpPost request not working

### DISCUSSION THREAD

**Discussion by minhdn2** [OrchardCMS/OrchardCore#16049] [Q&A]
Hi,

I have created an API module and it's not working with `HttpPost` request.
![image](https://user-images.githubusercontent.com/35803003/86440152-2b434780-bd34-11ea-89bf-be30723084d9.png)
![image](https://user-images.githubusercontent.com/35803003/86440197-3bf3bd80-bd34-11ea-80f7-0b939cab7c54.png)

If I change to `HttpGet`, the API works normally.

Please help to check and advise on how I can use `HttpPost` in Orchard Core.

Many thanks in advance.

**Comment by ns8482e** [comment]
It’s `Antiforgery`  see #6214
